### ISSUE TITLE
Working Copy Repository button doesn't open the correct Repository

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#113]
```
What steps will reproduce the problem?
1. Open Working Copy
2. Click Repository Button

What is the expected output? What do you see instead?
Should open the wc's svn repo.  Instead, it opens a random repo from my repo 
list.

What version of the product are you using? On what operating system?
1.3b1

Please provide any additional information below.


```

Original issue reported on code.google.com by `benmulli...@gmail.com` on 16 Sep 2010 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
Please provide more details, as it works perfectly for me.
What repo URL is shown in the WC window & what's the 'Path' field (from 
Repositories window) of the repo that is opened & the one you expect to be 
opened?
```

Original comment by `chris...@gmail.com` on 16 Sep 2010 at 11:36

**Comment by GoogleCodeExporter** [comment]
```
http://screencast.com/t/ZTAwNjQ1YjMt
http://screencast.com/t/NDM3YjI4OD
http://screencast.com/t/Y2Y5NTdhNjkt
http://screencast.com/t/MWZkZDM1NjM

Opens random repo from my repo list.  
Seems to cache that choice.  
Maybe related to old cache from previous version of svnx?
```

Original comment by `benmulli...@gmail.com` on 17 Sep 2010 at 12:20

**Comment by GoogleCodeExporter** [comment]
```
The repo shown in your 4th screen-shot _IS_ the one to which the URLs in 
screen-shots 1 & 3 refer.
However, it is the root of the repo & not the specific folder that the WC is 
for.
To get to your folder in the repo click 'trunk' (at the bottom) then 
'Clubhouse' (when it appears) & then double-click 'Cafe'.
I will see if this can be further improved.
```

Original comment by `chris...@gmail.com` on 17 Sep 2010 at 4:46

**Comment by GoogleCodeExporter** [comment]
```
Hmm, this is new behavior for this version.  I didn't see it doing that before.
Thanks for your time!
```

Original comment by `benmulli...@gmail.com` on 20 Sep 2010 at 11:24

**Comment by GoogleCodeExporter** [comment]
```
This should now be fixed/better in svnX 1.3.
```

Original comment by `chris...@gmail.com` on 2 Nov 2010 at 4:44
- Changed state: **Fixed**
